src/pages/tag.js

```diff
@@ -124,7 +124,7 @@ export default class TagGridPage extends Component {
       uiStore
     } = this.props;
     const pageSize = routingStore.query && routingStore.query.limit ? parseInt(routingStore.query.limit, 10) : uiStore.pageSize;
-    const sortBy = routingStore.query && routingStore.query.sortby ? routingStore.query.sortby : uiStore.sortBy;
+    const sortBy = routingStore.query && routingStore.query.sortby ? routingStore.query.sortby : "featured-desc";
```

Comment:
@mikemurray What's the best way to set `featured-desc` to be the default sort only for the Tag Listing Page, and to use the URL param or the SortBy selector otherwise?

Reply:
I believe `tag.js` is used for the tag pages so what you have above should work. What was `uiStore.sortBy` doing there, and was it needed for some reason?